android/quest/src/main/java/org/smartregister/fhircore/quest/data/report/measure/MeasureReportRepository.kt

```diff
@@ -83,6 +89,29 @@ constructor(
     val measureReport = mutableListOf<MeasureReport>()
     try {
       withContext(dispatcherProvider.io()) {
+        val measure =
+          fhirEngine
+            .search<Measure> { filter(Measure.URL, { value = measureUrl }) }
+            .first()
+            .resource
+
+        // TODO move to Sync
```

Comment:
We don't expect this to move to sync ever, we'll have to work with the 2 different ways of syncing fhir content